backend/internal/model/game.go

```diff
@@ -10,28 +13,37 @@ const (
 type Game interface {
 	AddGoal(int) error
 	ResetScore()
+	GetScore() GameScore
+	GetScoreChannel() chan GameScore
 	SubGoal(int) error
```

Comment:
Keep in mind, that in current implementation if there are no listeners on the channel, `SubGoal` and `AddGoal` will lock after 32 calls